demo/root/settings.py

```diff
@@ -229,6 +228,7 @@
     'SCHEME': 'https',
     # use default value, this will be overridden by value returned by BASE_LOCATION_GETTER
     'NETLOC': 'localhost',
+    'AUTH_CHECK_MIDDLEWARE': 'app.middleware.CustomSCIMAuthCheckMiddleware',
```

Comment:
Can we change this back to `app.middleware.SCIMAuthCheckMiddleware`. As a developer, I would want to name my class `CustomSCIMAuthCheckMiddleware` and have the library's version be either `SCIMAuthCheckMiddleware`, `BaseSCIMAuthCheckMiddleware`, etc.

Reply:
This *isn’t* the library’s version; it’s part of the demo app (`demo/app/middleware.py`), and has been since 0759c04182da3d182752c5f8362f97fb145c56b6. The only thing I changed here is to move it from `MIDDLEWARE_CLASSES` to `AUTH_CHECK_MIDDLEWARE`.

The library’s version is still `django_scim.middleware.SCIMAuthCheckMiddleware`, and developers who want that don’t need to customize `AUTH_CHECK_MIDDLEWARE` at all since it’s the default (`src/django_scim/settings.py`).

Reply:
Ah, yes.